index.html

```diff
@@ -4,7 +4,7 @@
         <body>
             <head>
                 <h1 id="title"> Welcome
-                    <p id="description"> this is our survey form
+                    <p id="description"> This is our survey form.
 
 <form id="survey-form">
     <input id="name"required> 
```

Comment:
Please note the <head>element is used to define the head portion of the document which contains information related to the document. The <head> tag contains other head elements such as <title>, <meta>, <link>, <style> <link> etc.
The correct tag would have been. <main> tag instead.